### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by jfernancordova**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ This alternative the easier to keep the plugin updated. Since the plugin is in t
 _See the [JetBrains Documentation](https://www.jetbrains.com/help/idea/managing-plugins.html?keymap=secondary_macos#repos) for more details._
 
 ### From disk
-This alternative allows you to choose the version ton install, also it avoids the Jetbrains moderation, so the last releases are available few days before than in the Marketplace.
+This alternative allows you to choose the version ton install, also it avoids the Jetbrains moderation, so the last releases are available in a few days before than in the Marketplace.
```
Comment (pcha) [inline README.md]: That's incorrect

File: README.md [hunk 2]
```diff
@@ -4,12 +4,12 @@ An IntelliJ-based IDEs plugin for [**Bloblang**](https://www.benthos.dev/docs/gu
 **Bloblang** is a  mapping language developed by @jeffail as part of [**Benthos**](https://github.com/benthosdev/benthos), a high performance and resilient stream processor.
 
 ## Features
-This pliugin recognizes the files `*.blobl` as bloblang files and check and highlight the syntax of these files. Fot more details tou can check the [CHANGELOG](CHANGELOG.md).
+This plugin recognizes the files `*.blobl` as bloblang files checking the highlight and syntax of these files. Fot more details you can check the [CHANGELOG](CHANGELOG.md).
```
Comment (pcha) [inline README.md]: The typos are ok, but the others changes aren't what I'ma trying to say. I think that the correct change is a comma after files

File: README.md [hunk 3]
```diff
@@ -4,12 +4,12 @@ An IntelliJ-based IDEs plugin for [**Bloblang**](https://www.benthos.dev/docs/gu
...
 
 ## Install
 ### From Marketplace
-This alternative the easier to keep the plugin updated. Since the plugin is in the very beginning of its development, it isn't in the stable channel but in the EAP. For installing plugins from this channel you have to add the repository to your plugins repositories list.
-1. Press `Ctrl+Alt+S` (or `⌘ ,` in Mac) to open the IDE settings and select **Plugins**.
+This alternative keeps the plugin updated. Since the plugin is in the beginning of its development, it isn't in the stable channel but in the EAP. For installing plugins from this channel you have to add the repository to your plugins repositories list.
```
Comment (pcha) [inline README.md]: Keep a plugin updated is a manual action, the IDE will only notify about updates. The word very is correct.

**Review by pcha — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -4,12 +4,12 @@ An IntelliJ-based IDEs plugin for [**Bloblang**](https://www.benthos.dev/docs/gu
...
+1. Press `Ctrl+Alt+S` (for Linux or Windows) (or `⌘ ,` for Mac) to open the IDE settings and select **Plugins**.
```
Comment (pcha) [inline README.md]: Press `Ctrl+Alt+S` for Linux and Windows or `⌘ ,` for Mac to open....

File: README.md (hunk 1 above)
Comment (jfernancordova) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (jfernancordova) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (jfernancordova) [reply README.md]: Fixed

File: README.md (hunk 4 above)
Comment (jfernancordova) [reply README.md]: Fixed

**Review by pcha — APPROVED** [review]

**Pull request merged**